samples/dnn/yolo_object_detection.cpp

```diff
@@ -1,3 +1,23 @@
+//! Brief Sample of using OpenCV dnn module in real time with device capture, video and image.
+//! Questions and suggestions email to: Alessandro de Oliveira Faria cabelo@opensuse.org or blog http://assuntonerd.com.br
```

Comment:
Please correct me if I wrong but you're not an author of this sample. Do you mind to replace it nearby a video reference?

Reply:
All right, I just wanted to help.